staging/src/k8s.io/kubectl/pkg/cmd/config/get_contexts.go

```diff
@@ -100,17 +107,7 @@ func (o *GetContextsOptions) Complete(cmd *cobra.Command, args []string) error {
 }
 
 // Validate ensures the of output format
-func (o *GetContextsOptions) Validate(cmd *cobra.Command) error {
-	validOutputTypes := sets.NewString("", "json", "yaml", "wide", "name", "custom-columns", "custom-columns-file", "go-template", "go-template-file", "jsonpath", "jsonpath-file")
-	supportedOutputTypes := sets.NewString("", "name")
-	outputFormat := cmdutil.GetFlagString(cmd, "output")
-	if !validOutputTypes.Has(outputFormat) {
```

Comment:
One line below, outputFormat is checked within supportedOutputTypes. It looks that this validOutputType check has no meaning.